### PR TITLE
feat: added 3 test that verify the pages load

### PR DESCRIPTION
**Pull request by Josue-SR**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: __tests__/pages/classes.test.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      useSession: jest.fn()
+    };
+});
+
+describe('Classes page tests', () => {
+    it('Classes page is loaded', async () => {
+        useSession.mockImplementation(() => {
+            return { data: { user: 'test@google.com' }, status: 'authenticated' };
+        });
+        const { container } = render(<Classes />);
```
Comment (roberthuskins) [inline __tests__/pages/classes.test.jsx:19]: most likely only need to render(<Classes>) if we are just checking if the page renders

File: __tests__/pages/classes.test.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    const originalModule = jest.requireActual('next-auth/react');
+    return {
+      __esModule: true,
+      ...originalModule,
+      useSession: jest.fn()
+    };
+});
+
+describe('Classes page tests', () => {
+    it('Classes page is loaded', async () => {
```
Comment (roberthuskins) [inline __tests__/pages/classes.test.jsx:15]: "Classes page can load"

**Comment by GuillermoFloresV** [comment]
We should refactor our main content for pages to become independently unit testable which would eliminate the need to check for a session when a page loads. This can be a separate issue/PR

File: __tests__/pages/join.test.jsx
Comment (GuillermoFloresV) [inline __tests__/pages/join.test.jsx:1]: Same here with the session not being mocked.

File: __tests__/pages/classes.test.jsx
Comment (GuillermoFloresV) [inline __tests__/pages/classes.test.jsx:1]: Is there a reason why we do not need to mock the session inside of the classes page?

**Comment by GuillermoFloresV** [comment]
Closed due to inactivity.

If you would still like to contribute, please resubmit this PR with the appropriate changes that were requested.

Thanks!

**Pull request closed without merging**